src/Altinn.Platform/Altinn.Platform.Storage/IntegrationTest/Clients/InstanceClient.cs

```diff
@@ -111,9 +116,10 @@ public async Task<Instance> PostInstances(string appId, int instanceOwnerPartyId
         /// <returns>the instance just created</returns>
         public async Task<Instance> PostInstances(string appId, Instance instanceTemplate)
         {
-            string requestUri = $"{versionPrefix}/instances?appId={appId}";
+            string requestUri = $"{_versionPrefix}/instances?appId={appId}";
 
-            HttpResponseMessage response = await client.PostAsync(hostName + requestUri, instanceTemplate.AsJson());
+            _client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", _validToken);
+            HttpResponseMessage response = await _client.PostAsync(_hostName + requestUri, instanceTemplate.AsJson());
```

Comment:
Consider using PostAsJsonAsync instead of calling .AsJson() at the end.

Reply:
Client will be obsolete once integration tests are updated, so won't fix this for now.